asn1/LedgerLedgerLotocol.asn

```diff
@@ -107,18 +107,27 @@ CALL ::= CLASS {
 CallSet CALL ::= {
     -- All requests MUST use odd typeIds
     -- All responses MUST use even typeIds
-    {1 PacketRequest} |
-    {2 PacketResponse} |
-    {3 PrepareRequest} |
-    {4 PrepareResponse} |
-    {5 FulfillRequest} |
-    {6 FulfillResponse} |
-    {7 RejectRequest} |
-    {8 RejectResponse} |
-    {9 TransferStateRequest} |
-    {10 TransferStateResponse} |
-    {11 CustomRequest} |
-    {12 CustomResponse}
+    -- {1 reserved because this is the typeId of InterledgerProtocolPaymentPacket in
+    -- https://github.com/interledger/rfcs/edit/mj-lll-packet-types/asn1/GenericPacket.asn } |
+    {2 QuoteLiquidityRequest} |
+    {3 QuoteLiquidityResponse} |
+    {4 QuoteBySourceAmountRequest} |
+    {5 QuoteBySourceAmountResponse} |
+    {6 QuoteByDestinationAmountRequest} |
+    {7 QuoteByDestinationAmountResponse} |
```

Comment:
Note that these would be the ILP packets as already defined, so with extensions, but without SideProtocol. This means you can't freely add fields, but only if the ILQP spec authors admit your custom extension as an official ASN1 extension for ILQP. They could, of course create a generic "SideProtocol" extension field, and that would then be an official extensions that allows ad-hoc extensions (IIUC).